draft-ietf-ppm-dap.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: "Privacy Preserving Measurement"
-docname: draft-gpew-priv-ppm-latest
+title: "Distributed Aggregation Protocol for Privacy Preserving Measurement"
```

Comment:
I believe you're also going to want
```suggestion
title: "Distributed Aggregation Protocol for Privacy Preserving Measurement"
abbrev: DAP-PPM
```
Other possibilities: "PPM-DAP", "Distributed Aggregation Protocol".